Sources/Transitions/Transitions.swift

```diff
@@ -0,0 +1,9 @@
+// Copyright © 2019 SkeletonView. All rights reserved.
+
+import UIKit
+
+
+public enum SkeletonTransitionStyle {
+    case none
+    case fade(duration: TimeInterval)
```

Comment:
we don't need an alias, TimeInterval is enough descriptive to know that means time. What do you think?